web/src/message_list_view.js

```diff
@@ -803,6 +798,19 @@ export class MessageListView {
         );
     }
 
+    set_edited_notice_locations(message_container) {
+        // This function sets the edit notices of message_container such as:
+        // `message_edit_notices_in_left_col`, `message_edit_notices_alongside_sender`,
+        // and `message_edit_notices_for_status_message`.
```

Comment:
This comment isn't very helpful. I think we can just say "Based on the variables the define the overall message's HTML layout, set variables defining where the message-edited notices should be placed."